src/uid2StorageManager.ts

```diff
@@ -0,0 +1,47 @@
+import { UID2 } from "./uid2Sdk";
+import { isValidIdentity, Uid2Identity } from "./Uid2Identity";
```

Comment:
nit: isValidIdentity and UID2 is not used

Reply:
removed!